docs/reference/formulas/smartwatts.md

```diff
@@ -36,33 +33,46 @@ You can use the following command to install SmartWatts:
 
 For running the SmartWatts Formula you need: a Source and a Destination, a Sensor that provides `HWPCReports` and a configuration.
 
-### Source and Destination
-For running SmartWatts we are using MongoDB as Source and InfluxDB 2.X as Destination as dockers containers.
+### Input and Output
+As any Formula, SmartWatts needs both inputs and outputs.
+We can choose those among [this list](../database/sources_destinations.md#summary)
 
```

Comment:
I would say that in the lunching exemple below, we are using mongo as input and influx as source